### PR TITLE
CP-41964:Templates modification for guest support becomes EOL, Deprecated or Official in XenServer

### PR DESCRIPTION
**Pull request by MinghuiHu**
Test Execution
https://info.citrite.net/pages/viewpage.action?pageId=1625982439

### CODE REVIEW

**Comment by xihuan-citrix** [comment]
Hi @rosslagerwall , @MinghuiHu and I both could not add reviewer for this PR. Could you help to add @xihuan-citrix  and you  as the reviewer list.

**Review by acefei — APPROVED** [review]

**Review by rosslagerwall — APPROVED** [review]

File: json/debian-11.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "uuid": "acfeecf4-4007-412b-8f13-6d61a6aee5bd",
     "reference_label": "debian-11",
-    "name_label": "Debian Bullseye 11 (experimental)",
+    "name_label": "Debian Bullseye 11",
```
Comment (stormi) [inline json/debian-11.json:4]: This change doesn't match the commit message, does it?

Comment (xihuan-citrix) [reply json/debian-11.json:4]: Hi @MinghuiHu , Could change the message to 
```
CP-41964:Templates modification for guest support becomes EOL, Deprecated or Official in XenServer
* Remove CoreOS template for EOL
* Add deprecated label for sles124 and ubuntu1804's templates
* Remove experimental label from debian11's template
```

Comment (MinghuiHu) [reply json/debian-11.json:4]: I have refined the commit messages detailly

**Review by xihuan-citrix — APPROVED** [review]

**Pull request merged**